src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSFilterArgumentCollection.java

```diff
@@ -190,6 +193,16 @@
             optional = true)
     public int minIdentity = 30;
 
+    /**
+     * If using --is-host-aligned, ignores alignments to these contigs (can be specified multiple times). This
+     * can be useful if the alignment is to a reference containing a microbe, such as chrEBV in hg38.
```

Comment:
Is it really called chrEBV?  Honestly I haven't looked at all 3500 names.

Reply:
Yep. I recently found out it's also in hg19 but it uses the NCBI accession